beta/src/pages/learn/render-and-commit.md

```diff
@@ -65,28 +65,28 @@ export default function Image() {
 
 </Sandpack>
 
-Try commenting out the `ReactDOM.render` call and see the component disappear!
+`ReactDOM.render` कॉल पर कमेंट करने का प्रयास करें और देखें कि कौम्पोनॅन्ट गायब हो गया है|
 
-### Re-renders when state updates {/*re-renders-when-state-updates*/}
+### स्टेट अपडेट होने पर पुन: रेंडर {/*re-renders-when-state-updates*/}
```

Comment:
`स्टेट` => `state`

I think i missed this. but we are not translating `state`

Can you find and replace this in whole file

Reply:
Replaced state in the entire file.